### PR TITLE
Vendor onig WASM dependency

### LINKED ISSUE #538: Vendor the oniguruma wasm file
Currently it is not possible to vendor the oniguruma wasm needed by `starry-night` ourselves.

This issue is to track the progress.

References:
https://github.com/wooorm/starry-night/issues/8

### PR DESCRIPTION
**Pull request by sebastinez**
Instead of letting `starry-night` fetch the `vscode-oniguruma` WASM, we fetch it from the `node_modules` folder and do the same in the e2e tests.
For this to work we need to update `starry-night` to version `1.5.0`

Closes #538

### CODE REVIEW

File: tests/support/fixtures.ts [hunk 1]
```diff
@@ -87,8 +92,7 @@ export const test = base.extend<{
           route.request().url().startsWith("https://esm.sh/vscode-oniguruma@1")
         ) {
           route.fulfill({
-            status: 200,
-            path: "./tests/support/onig.wasm",
+            path: onigWASMPath,
```
Comment (rudolfs) [inline tests/support/fixtures.ts:95]: Do we even need any of this in the test file? Should it not just work now that we load it in the `syntax.ts` file?

Comment (sebastinez) [reply tests/support/fixtures.ts]: Yeah you're right, we don't need this anymore.
8551ab4 removes it

**Comment by rudolfs** [comment]
The tests fail when I run them locally because there's a console error. Any idea why that is?

Screenshot 2023-01-05 at 14 24 28

```
  16 failed
    [chromium] › hashRouter.spec.ts:78:3 › project page navigation › navigate project paths ========
    [chromium] › hashRouter.spec.ts:98:3 › project page navigation › navigate project paths with a selected peer
    [chromium] › historyRouter.spec.ts:72:3 › project page navigation › navigate project paths =====
    [chromium] › historyRouter.spec.ts:92:3 › project page navigation › navigate project paths with a selected peer
    [chromium] › project/commit.spec.ts:103:1 › navigation to source tree at specific revision =====
    [chromium] › project.spec.ts:25:1 › navigate to project ========================================
    [chromium] › project.spec.ts:75:1 › source file highlighting ===================================
    [chromium] › project.spec.ts:87:1 › navigate line numbers ======================================
    [chromium] › project.spec.ts:111:1 › navigate deep file hierarchies ============================
    [chromium] › project.spec.ts:154:1 › files with special characters in the filename =============
    [chromium] › project.spec.ts:198:1 › binary files ==============================================
    [chromium] › project.spec.ts:207:1 › hidden files ==============================================
    [chromium] › project.spec.ts:215:1 › markdown files ============================================
    [chromium] › project.spec.ts:253:1 › peer and branch switching =================================
    [chromium] › project.spec.ts:428:3 › browser error handling › error appears when a image with a relative path can't be loaded
    [chromium] › settings.spec.ts:35:1 › change theme ==============================================
```

**Comment by sebastinez** [comment]
Yeah I think that's due to your `node_modules` folder, not having the latest version of `starry-night`, try to.

```
rm -rf node_modules
npm cache clean --force
npm install
```

it should work then.

**Comment by rudolfs** [comment]
> Yeah I think that's due to your `node_modules` folder, not having the latest version of `starry-night`, try to.

Nice, that fixed it.

**Review by rudolfs — APPROVED** [review]
Nice work! 👍

**Pull request merged**